build.gradle

```diff
@@ -323,6 +323,9 @@ dependencies {
 
   implementation group: 'com.networknt', name: 'json-schema-validator', version: versions.jsonSchemaValidator
 
+  implementation group: 'com.azure', name: 'azure-core', version: '1.26.0'
```

Comment:
Could we also do the same version thing for these so version.azureCore and version.azureMessagingServiceBus will look cleaner!